.github/actions/verify-token/src/index.ts

```diff
@@ -62,11 +62,22 @@ async function run(): Promise<void> {
     if (!ghToken) {
       throw new Error("token not provided");
     }
+    const builderInterfaceType = core.getInput("builder-interface-type");
```

Comment:
You can pass a required option to get input and it will throw an error if it wasn't supplied

Reply:
done, thanks